### DISCUSSION TITLE
Help for first time user

### DISCUSSION THREAD

**Discussion by Bwangster12** [TheLastGimbus/GooglePhotosTakeoutHelper#177] [I NEED HELP] [answered]
I did download all my photos, user created albums and albums with years.  Is there a way to use this on all photos?

It seems I have to click on one album at a time in the tool and not the entire Google Photos directory.

Also, I tested this on a single album folder and there was a lot of photos that had been custom named in GP that it didn't process.  I do see the corresponding json in there.

**Comment by TheLastGimbus** [comment]
1. Read "How to use" section of readme, and get your takeout as it says
2. Run gpth by double clicking and follow instructions

**Reply by Bwangster12** [reply]
So if I downloaded everything because I want to keep the custom folders I created I can't use this on those?  This only works on the Photos from XXXX folders?

I had a lot of photos in one of those folders that I had renamed years ago.  The JSON are all in there matching, but it gave errors for all of those.

**Reply by TheLastGimbus** [reply]
Oh, i think i get it, that's interesting

Could you send me a screenshot of those jpg/json files / their names etc? Also the json files (their contents) 

If i could look at the full picture i could resolve this

> This only works on the Photos from XXXX folders?

Yes, but all photos from albums are in year folders anyways

 - plus, i could finally implement those albums - that's a "todo" for a while - look at #135

**Comment by TheLastGimbus** [comment, marked as answer]
Albums are already implemented :tada: #187
